src/validators/tasks.py

```diff
@@ -29,22 +30,27 @@
     update_unused_validator_keys_metric,
 )
 from src.validators.keystores.base import BaseKeystore
+from src.validators.keystores.local import LocalKeystore
 from src.validators.signing.common import get_validators_proof
 from src.validators.typings import (
     ApprovalRequest,
     DepositData,
     NetworkValidator,
     Validator,
+    ValidatorsRegistrationMode,
 )
 from src.validators.utils import send_approval_requests
 
 logger = logging.getLogger(__name__)
 
 
+pending_validator_registrations: list[HexStr] = []
```

Comment:
Would that work if I run with `--pool-size=2`?

Reply:
`pool-size` affects only keystores loading. It is not server concurrency. Server works in a single thread. Thread safety is not necessary for now.

Reply:
pool-size affects only keystores loading. It is not server concurrency.

Reply:
I removed pool-size from start-api command args